new/prusti/tests/pass/parse/implies_fail.stderr

```diff
@@ -0,0 +1,20 @@
+error: expected Prusti assertion
+  --> $DIR/implies_fail.rs:10:17
+   |
+10 | #[requires(true ==>)]
+   |                 ^^^
+
+error: expected Prusti assertion
+  --> $DIR/implies_fail.rs:13:12
+   |
+13 | #[requires(==> true)]
+   |            ^^^
+
+error: expected expression
```

Comment:
```suggestion
error: expected expression (an implication cannot be inside a Rust expression)
```

Reply:
this is not my error, this is the default error from `syn::parse`; is there a way to change it?